package.json

```diff
@@ -18,41 +18,55 @@
   "license": "MIT",
   "author": "Isaac J. Lee",
   "type": "module",
-  "main": "src/index.js",
-  "bin": "./bin/ember-codemod-v1-to-v2.js",
+  "main": "dist/src/index.js",
+  "bin": "dist/bin/ember-codemod-v1-to-v2.js",
   "directories": {
     "test": "tests"
   },
   "files": [
-    "bin/",
-    "src/"
+    "dist"
   ],
   "scripts": {
+    "build": "tsc --project tsconfig.build.json && chmod +x dist/bin/ember-codemod-v1-to-v2.js && cp -r blueprints dist/blueprints",
```

Comment:
The `build` and `test` scripts became more complex, especially due to the presence of blueprint files.

Since TypeScript includes `src`, I don't think there's a way to not check `src/blueprints` for types, but still include ("grandfather") the files for compilation. We shouldn't add `// @ts-nocheck` to every blueprint file, because it is inefficient and alters the output of a blueprint file.

For simplicity, I decided to move the `src/blueprints` folder to `blueprints` at the project root level. I think this is the approach that `emberjs/ember.js` takes:

<img width="300" alt="" src="https://github.com/ijlee2/ember-codemod-v1-to-v2/assets/16869656/e4be5986-f8ca-4a89-a86e-a2e0424a1143">

Note that its [`tsconfig.json`](https://github.com/emberjs/ember.js/blob/v5.2.0-alpha.3/tsconfig.json) does not refer to `blueprints` or `blueprints-js`.